packages/generators/init-template/react/webpack.config.js.tpl

```diff
@@ -0,0 +1,105 @@
+<% if (isCSS) { %>
+<% if (extractPlugin === "Yes") { %>
+const stylesHandler = MiniCssExtractPlugin.loader;
+<% } else if (extractPlugin === "Only for Production") { %>
+const stylesHandler = isProduction ? MiniCssExtractPlugin.loader : 'style-loader';
+<% } else { %>
+const stylesHandler = 'style-loader';
+<% } %>
+<% } %>
+
+const config = {
+    entry: '<%= entry %>',
+    output: {
+        path: path.resolve(__dirname, 'dist'),
+    },<% if (devServer) { %>
+    devServer: {
+        open: true,
+        host: 'localhost',
+    },<% } %>
+    plugins: [<% if (htmlWebpackPlugin) { %>
+        new HtmlWebpackPlugin({
+            template: 'index.html',
+        }),
+<% } %><% if (extractPlugin === "Yes") { %>
+        new MiniCssExtractPlugin(),
+<% } %>
+        // Add your plugins here
+        // Learn more about plugins from https://webpack.js.org/configuration/plugins/
+    ],
+    module: {
+        rules: [<% if (langType == "ES6") { %>
+            {
+                test: /\.?js$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: "babel-loader",
+                    options: {
+                        presets: ["@babel/preset-env", "@babel/preset-react"],
+                    },
+                },
+            },<% } %><% if (langType == "Typescript") { %>
+            {
+                test: /\.(ts|tsx)$/i,
+                loader: 'ts-loader',
+                exclude: ['/node_modules/'],
+            },<% } %><%  if (isCSS && !isPostCSS) { %>
+            {
+                test: /\.css$/i,
+                use: [stylesHandler,'css-loader'],
+            },<% } %><%  if (cssType == 'SASS') { %>
+            {
+                test: /\.s[ac]ss$/i,
+                use: [stylesHandler, 'css-loader', <% if (isPostCSS) { %>'postcss-loader', <% } %>'sass-loader'],
+            },<% } %><%  if (cssType == 'LESS') { %>
+            {
+                test: /\.less$/i,
+                use: [<% if (isPostCSS) { %>stylesHandler, 'css-loader', 'postcss-loader', <% } %>'less-loader'],
+            },<% } %><%  if (cssType == 'Stylus') { %>
+            {
+                test: /\.styl$/i,
+                use: [<% if (isPostCSS) { %>stylesHandler, 'css-loader', 'postcss-loader', <% } %>'stylus-loader'],
+            },<% } %><%  if (isPostCSS && isCSS) { %>
+            {
+                test: /\.css$/i,
+                use: [stylesHandler, 'css-loader', 'postcss-loader'],
+            },<% } %>
+            {
+                test: /\.(eot|svg|ttf|woff|woff2|png|jpg|gif)$/i,
+                type: 'asset',
+            },
+
+            // Add your rules for custom modules here
+            // Learn more about loaders from https://webpack.js.org/loaders/
+        ],
+    },
+    resolve: {
+        alias: {
+            "@": path.resolve(__dirname, "./src/"),
+        },<% if (langType == "Typescript") {%>
+        extensions: ['.tsx', '.ts', '.js'],<% } %>
```

Comment:
Let's use `['.tsx', '.ts', '.jsx', '.js', '...']`, we should use `...` everythere

Reply:
Okay 👍🏽